### ISSUE TITLE
Website: make competitors editable

### ISSUE THREAD

**Issue by Xilaew** [Xilaew/JudoShiai-xPack#12]
The Hint box encouraging people to create a trainer id states that they can modify registered competitors later on, but this feature is not implemented.

**Comment by plysakcz** [comment]
is there a way to implemented then?

**Comment by Xilaew** [comment]
Yes there is. If you have implemented it you can send a pull request, and i
will merge it. Or you tell me about you and why this feature is important
to you and I might implement it in the next weeks.

Cheers Felix
--
Felix von Poblotzki Geb. Kurth
Informatik-Ingenieur M.Sc
telefon: +49 152 32787790
email: ***@***.***
website: xilaew.eu

plysakcz ***@***.***> schrieb am Mi., 20. Okt. 2021, 02:06:
